src/main/java/com/direwolf20/mininggadgets/client/renderer/MiningGadgetModel.java

```diff
@@ -0,0 +1,37 @@
+package com.direwolf20.mininggadgets.client.renderer;
+
+import net.minecraft.client.renderer.model.IBakedModel;
+import net.minecraft.client.renderer.model.ItemCameraTransforms;
+import net.minecraftforge.client.event.ModelBakeEvent;
+import org.apache.commons.lang3.tuple.Pair;
+
+import javax.vecmath.Matrix4f;
+import java.util.Arrays;
+import java.util.List;
+
+public class MiningGadgetModel extends CustomRenderItemBakedModel
```

Comment:
Why add 3 classes for this, instead of putting all into one? It's unlikely that MG is going to need the other ones for something else in the future - even though I like the seperation of concerns created by this, it's propably more complex then needed...